src/TextUI/Command.php

```diff
@@ -1347,6 +1346,8 @@ private function handleOrderByOption(string $value): void
                     $this->arguments['resolveDependencies'] = true;
 
                     break;
+                default:
+                    $this->exitWithErrorMessage("unrecognized order-by option -- $order");
```

Comment:
why double `--` ?

Reply:
Fixed. Was a leftover from where I found this in `GetOpt`.